src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/FSPath.kt

```diff
@@ -35,9 +36,21 @@ class FSPath(val path: Path, val fs: FileSystem) {
     fun resolve(child: String): FSPath =
         FSPath(path.resolve(child), fs)
 
+    operator fun div(child: String): FSPath =
+        this.resolve(child)
+
     fun source(): Source =
         fs.source(path)
 
+    fun readUtf8(): String =
+        source().buffer().readUtf8()
```

Comment:
issue: the `readUtf8` method should called inside `use` block to make sure that resource is closed. It is not critical here since we are not opening a real resource but it still would be better to close resource correctly

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/372/commits/0bb98cac5aa21ca07e27049f71c33cbaa0789950